docs/release-notes-v14.0.md

```diff
@@ -0,0 +1,83 @@
+
+!!! seealso
+
+    * [PostgreSQL 14 release notes](https://www.postgresql.org/docs/release/14/)
+    * Percona Blog: [PostgreSQL 14 – Performance, Security, Usability, and Observability](https://www.percona.com/blog/postgresql-14-performance-security-usability-and-observability/)
+
+The following is the list of extensions available in Percona Distribution for PostgreSQL.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.0 | a HA (High Availability) solution for PostgreSQL |
+| [Pgaudit](https://www.pgaudit.org/)             | 1.6.0 Beta | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+|[`pgAudit set user`](https://github.com/pgaudit/set_user)| 2.0.1|  provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.34    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 11.5       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.15.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.6   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 0.9.2 - Beta1 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
+|[`wal2json`](https://github.com/eulerto/wal2json) |2.3        | a PostgreSQL logical decoding JSON output plugin.|
+
+Percona Distribution for PostgreSQL also includes the ETCD packages which are used for Patroni cluster setup. These packages are available for the following operating systems:
+
+|  Operating System |Package               | Description                  |
+| ------------------- | ---------------------| ---------------------------- |
+| CentOS 7            |`python3-python-etcd` | A Python client for ETCD     |
+| CentOS 8            | `etcd`               | A consistent, distributed key-value store|
+|                     | `python3-python-etcd`| A Python client for ETCD     |
+| Debian 9 ('stretch')| `etcd`               | A consistent, distributed key-value store|
+|                     | `python3-etcd`       | A Python client for ETCD     |
+
+                                                      
+Percona Distribution for PostgreSQL is also shipped with the [libpq](https://www.postgresql.org/docs/14/libpq.html) library. It contains "a set of
+library functions that allow client programs to pass queries to the PostgreSQL
+backend server and to receive the results of these queries." [^1]
+
+
+[^1]: https://www.postgresql.org/docs/13/libpq.html
```

Comment:
I think that footnote can be removed? It still points to version 13 of the docs, and we have a link in the paragraph above.

Reply:
Removed